embedly/embedly/extract.py

```diff
@@ -23,7 +27,12 @@ def _get_cached_url(self, url):
         cached_data = self.redis_client.get(cache_key)
 
         if cached_data is not None:
-            return json.loads(cached_data)
+            try:
+                return json.loads(cached_data)
+            except ValueError:
+                raise URLExtractorException(
```

Comment:
This is a robustness feature that should be added later. A blanket failure is appropriate right now, as a first iteration IMO.